### ISSUE TITLE
Unmet Peer Dependency: react-native (fresh install of newest react-native)

### ISSUE THREAD

**Issue by julianh2o** [react-native-community/react-native-navbar#173]
```
$ npm install --save react-native-navbar
MyApp@0.0.1 /path/to/MyApp
├── UNMET PEER DEPENDENCY react-native@0.27.0-rc2
└── react-native-navbar@1.5.0

npm WARN react-native-navbar@1.5.0 requires a peer of react-native@>= 0.16 but none was installed.
```

It seems to me that I'm using a valid version, I'm a bit confused why I'm getting this error..

```
$ react-native --version
react-native-cli: 0.2.0
react-native: 0.27.0-rc2
```

I was able to get things working by running `npm install react-native` and then re-running the command. But I think that most people will probably have react-native installed globally (which seems to be what was causing the problem?)

**Comment by zhenfdfs** [comment]
same problerm

**Comment by waleedarshad-vf** [comment]
same problerm

**Comment by Ellegarden1219** [comment]
same problem

**Comment by Kureev** [comment]
Just released 1.6.3 where peerDependencies has been removed
